RecoLocalCalo/HGCalRecProducers/plugins/alpaka/ConstantsForClusters.h

```diff
@@ -0,0 +1,10 @@
+#ifndef RecoLocalCalo_HGCalRecProducers_interface_alpaka_ConstantsForClusters_h
+#define RecoLocalCalo_HGCalRecProducers_interface_alpaka_ConstantsForClusters_h
+
+namespace constants {
```

Comment:
```suggestion
namespace hgcal::constants {
```
?

For example, we have the namespace `hcal::constants`, _etc_.